conans/test/functional/python_requires/python_requires_test.py

```diff
@@ -576,6 +576,36 @@ class Lib(ConanFile):
         self.assertIn("Same python_requires with different versions not allowed for a conanfile",
                       t.out)
 
+    def short_paths_test(self):
+        # https://github.com/conan-io/conan/issues/5814
+        client = TestClient(servers={"default": TestServer()},
+                            users={"default": [("lasote", "mypass")]})
+        conanfile = """from conans import ConanFile
```

Comment:
Keep it simple, remove unused things:
- license, author, generator

Use textwrap.dedent() for cleaner layout